hooks/validator/README.md

```diff
@@ -0,0 +1,86 @@
+# Validator Hook
+
+The `validator hook` taps into the `After` lifecycle event to validate the result of flag evaluations.
```

Comment:
@skyerus @beeme1mr think it's worth mentioning right here that it defaults if it's invalid?

Maybe that's obvious and takes away from this otherwise very snappy sentence. :shrug:

Reply:
Done!